docs/UserGuide.md

```diff
@@ -73,75 +73,34 @@ The following annotated text-boxes are used throught this guide to provide usefu
 
 ## Quick Start
 
-In this section, lets learn how to set up MediBase3 on your computer.
+In this section, let's learn how to set up MediBase3 on your computer.
 
 ### Installing MediBase3
 
-#### Step 1: Install JDK 17
+#### Step 1: Check if you have Java 17 installed your system.
```

Comment:
"Check if you have Java 17 installed on your system." Missing "on"

Reply:
**Java17**:
![image](https://github.com/user-attachments/assets/d397ae7d-ab9f-45d7-8713-a506c526df1e)